src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -236,6 +237,32 @@ public void parseRecognizesEntryWithUnknownType() throws IOException {
     }
 
     @Test
+    public void parseReallyUnknownType() throws Exception {
+        String bibtexEntry = "@ReallyUnknownType{test," + OS.NEWLINE +
+                " Comment                  = {testentry}" + OS.NEWLINE +
+                "}";
+
+        Collection<BibEntry> entries = new BibtexParser(importFormatPreferences).parseEntries(bibtexEntry);
+        BibEntry entry = entries.iterator().next();
+
+        String resourceName = "/testbib/reallyunknowntype.bib";
```

Comment:
I think this is not necessary in this case. If you really want to test it, then please use the Resources directory handling, like in the` BibTexImportetTest.class ` and do not hardcode the relative dir. 
` Paths.get(BibtexImporterTest.class.getResource("BibtexImporter.examples.bib").toURI());`